contrib/opencensus-ext-azure/README.rst

```diff
@@ -37,11 +37,11 @@ This example shows how to send a warning level log to Azure Monitor.
     logger.addHandler(AzureLogHandler(connection_string='InstrumentationKey=<your-instrumentation_key-here>'))
     logger.warning('Hello, World!')
 
-* Alternatively, you can specify your `connection string` in an environment variable ``APPLICATIONINSIGHTS_CONNECTION_STRING``.
```

Comment:
Is not possible to use the environment variable anymore?

Reply:
It is, this was just a duplicated line in the docs.